### PR TITLE
{tools}[dummy/dummy] MLC v3.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0aa16ef88557f3ffef2c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/113a1fd96d78cabb4e87 for a full test report.

**Comment by boegel** [comment]
@wpoely86: ...

File: easybuild/easyconfigs/m/MLC/MLC-3.0.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ and how they change with increasing load on the system."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# download requires agreeing to license
+# https://software.intel.com/en-us/articles/intelr-memory-latency-checker#download
+sources = ['mlcv3.0.tgz']
+
+sanity_check_paths = {
+    'files': ['Linux/mlc', 'Linux/mlc_avx512', 'mlc_license.txt', 'readme.pdf'],
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MLC/MLC-3.0.eb]: do we really want a license and readme in the sanity check?

Comment (boegel) [inline easybuild/easyconfigs/m/MLC/MLC-3.0.eb]: well, they're part of the tarball being unpacked... if they're not there, soemthing is wrong

and we do this in other places too, license and documentation are important part of an installation?

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

**Pull request merged**